site/_resources/upgrading.md

```diff
@@ -19,17 +19,84 @@ You can also read the <a href="{% link _guides/ingressroute-to-httpproxy.md %}">
 </div>
 
 <div class="alert-deprecation">
-<b>Deprecation Notice</b><br>
+<b>Deprecation Notices</b><br>
 The <code>request-timeout</code> field in the Contour config file is deprecated as of Contour 1.7 and has been replaced by the <code>timeouts.request-timeout</code> field.
 The deprecated field will be removed in a future release.
 Please see the documentation for <a href="{% link docs/{{site.latest}}/configuration.md %}">the Contour config file</a>.
+
+Contour annotations starting with `contour.heptio.com` have been removed from documentation for some time. Contour 1.8 marks the official deprecation of these annotations, to be removed in a future release. These annotations will now result in warning error logs when used.
```

Comment:
Nit: either hard-wrap the line, or do a sentence per line :)

"warning error logs" reads awkwardly, recommend something like "Use of these annotations will be logged as warnings"